src/routes/lnurl/components/withdraw.svelte

```diff
@@ -0,0 +1,2 @@
+<script lang="ts">
```

Comment:
required component? or can we delete?

Reply:
This is a placeholder for the withdraw LNURL feature which I am doing next, so I think it is fine to leave in for the moment since it is not imported.